cmd/vulcan-exposed-rdp/main.go

```diff
@@ -121,12 +122,26 @@ func isExposedRDP(host, port string, timeout int) []report.Vulnerability {
 
 	if CCPDURegex.Match(b) && b[5] == byte('\xd0') {
 		exposedRDP.Details += fmt.Sprintf("RDP host detected in port: %s", port)
+		exposedRDP.AffectedResource = fmt.Sprintf("%s", port)
+		exposedRDP.ID = computeVulnerabilityID(host, exposedRDP.AffectedResource, port)
+		exposedRDP.Labels = []string{"issue", "rdp"}
 		return []report.Vulnerability{exposedRDP}
 	}
 
 	return nil
 }
 
+func computeVulnerabilityID(target, affectedResource string, elems ...interface{}) string {
```

Comment:
This function is now available from the SDK as: ```helpers.ComputeFingerprint(elems ...interface{}) string```

Reply:
Alright, using the SDK function now.